scripts/lint-mdc.ts

```diff
@@ -80,16 +83,24 @@ interface PackageJson {
 
 const PACKAGES_DIR_ABSOLUTE = path.resolve(__dirname, '../packages');
 
+// TODO(acdvorak): Remove Corner, CornerBit, ALWAYS_FLOAT_TYPES, VALIDATION_ATTR_WHITELIST
```

Comment:
When you say remove... do you mean these should be mentioned in the READMEs so they don't need to be excluded here?